custom_components/anniversaries/sensor.py

```diff
@@ -1,24 +1,18 @@
 """ sensor """
 
 from dateutil.relativedelta import relativedelta
-from datetime import datetime, date, timedelta
+from datetime import datetime, date
 
 import logging
 from homeassistant.helpers.entity import Entity, generate_entity_id
-from homeassistant.core import HomeAssistant, State
 from homeassistant.components.sensor import ENTITY_ID_FORMAT
 
 from homeassistant.const import (
     CONF_NAME,
 )
 
 from .const import (
-    ATTRIBUTION,

```

Comment:
please keep `ATTRIBUTION`  It is going to be used in a future version

Reply:
https://github.com/pinkywafer/Anniversaries/pull/79/commits/736300615cfeb1bb72f1127ac4594f1160d3714f